python/n.py

```diff
@@ -83,6 +84,15 @@ def type_check(file, tree):
 	return (error_count, warning_count)
 
 async def parse_tree(tree):
+	running = True
+	# HACK: `anger` does nothing but wait, but since it begins a new sleep every
+	# second, when there's a keyboardinterrupt it can (ungracefully) explode
+	# instead of ignoring it.
```

Comment:
Surely theres a better way to do this?

Reply:
I realized it is due to

```py
# https://github.com/encode/httpx/issues/914#issuecomment-622586610
if sys.version_info[0] == 3 and sys.version_info[1] >= 8 and sys.platform.startswith('win'):
    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

which is needed to prevent aiohttp from being like "RuntimeError: Event loop is closed owo" or something